src/Java.Interop.Dynamic/Tests/Java.Interop.Dynamic-Tests.csproj

```diff
@@ -36,9 +36,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\lib\NUnit-2.6.3\bin\nunit.framework.dll</HintPath>
```

Comment:
From the commit message:

> the latter one is deprecated as of Mono 4.9.

I guess the problem is that `nunit.framework.dll` *was* present in mono, and in mono 4.9 it isn't anymore, which is why this used to work but now it doesn't.